### ISSUE TITLE
What does input label mean in code?

### ISSUE THREAD

**Issue by ghm666** [yformer/EfficientSAM#39]
`input_points = torch.tensor([[[[580, 350], [650, 350]]]])
input_labels = torch.tensor([[[1, 1]]])

redicted_logits, predicted_iou = model(
        sample_image_tensor[None, ...],
        input_points,
        input_labels,
    )
`

**Comment by ghm666** [comment]
I get it, the label represents foreground and background

**Comment by xinli2008** [comment]
I have a question to bother you! If i want to do salienct segmentation in a arbitrary image, how should i set this parameter to fit random images?
